index.bs

```diff
@@ -152,33 +152,36 @@ When asked to <dfn abstract-op>update the Client Hints set</dfn> given a |settin
  <li>[=Add a new Accept-CH cache entry=] with |response|'s [=url/origin=] and |settingsObject|'s [=environment settings object/client hints set=].
 </ol>
 
-<dfn>Accept-CH state</dfn> (`name="accept-ch"`) {#accept-ch-state-algo}
+<dfn>Delegate-CH</dfn> {#delegate-ch-algo}
 --------
 
-Note: This metadata *appends* [=client hints token=]s to the [=environment settings object=]'s [=environment settings object/client hints set=]. It *does not* add those hints to the [=Accept-CH cache=]. After this algorithm runs once, no further modification to the [=environment settings object/client hints set=] can occur without reloading the page.
+Note: This metadata *appends* [=client hints token=]s to the [=environment settings object=]'s [=environment settings object/client hints set=]. It *does not* add those hints to the [=Accept-CH cache=].
 
 <ol>
  <li>Let |metaElement| be the <{meta}> element.
- <li>If |metaElement| is not a child of a <{head}> element, then return.
+ <li>If any <{link}> or <{script}> elements have begun to execute, then return.
```

Comment:
What about `<style>`? Should we also exclude it here?

Can you add an inline issue RE #110?